crates/bevy_ptr/src/lib.rs

```diff
@@ -557,6 +569,39 @@ impl<'a, T> From<&'a [T]> for ThinSlicePtr<'a, T> {
     }
 }
 
+/// A newtype around [`NonNull`] that only allows conversion to read-only borrows or pointers.
+///
+/// This type can be thought of as the `*const T` to [`NonNull<T>`]'s `*mut T`.
+#[repr(transparent)]
+pub struct RemoteDropPtr<T: ?Sized>(ConstNonNull<T>);
```

Comment:
This documentation looks wrong, `RemoteDropPtr` also drops the `T` it points to.

And unless I'm missing something this is unused?

Reply:
This has been removed in favor of `MovingPtr`.